build_config/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Support for parsing `build.yaml` configuration.
 repository: https://github.com/dart-lang/build/tree/master/build_config
```

Comment:
We should set the version here to be `0.4.6-dev` right?

Reply:
not necessarily.   this one may not need to be republished right away.

Reply:
Sure. But we want to make sure (at a glance) we know there are changes since the last stable publish. That's what the bump + -dev does

Reply:
For more info see https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change

Reply:
There are actually no changes here that should not be reverted when other packages are published.  I may add some later, but haven't yet.  (The sdk version change was erroneous and reverted).